### DISCUSSION TITLE
Save archive title option and additional titles

### DISCUSSION THREAD

**Discussion by darkred3** [Difegue/LANraragi#404] [Ideas]
Currently we've got the option to have LRR automatically convert the title to something more compact, as well as just using the Default Archive Title included with the json file or whatever is found on e-hentai/chaika/nhentai. Another option I found only within the e-hentai script is to use the original Japanese Title instead of the Default one.
I suggest leaving that parsed Compact Title out altogether while automatically saving **both** the Default Title and Japanese Title. This would concern all metadata scripts.
Additionally I also suggest a new option to switch between these two titles similar to what sadpanda has.

![sadpanda](https://user-images.githubusercontent.com/33666486/107857163-0025e800-6e2d-11eb-92f6-2e1693901b41.jpg)

**Comment by Difegue** [comment]
Not all metadata sources have dual titles available. Adding an extra field for alternate titles is doable, but I wonder if it'd really see much use.  

Moving this to a discussion.
